### ISSUE TITLE
Duplicated models JPER/OCTOPUS

### ISSUE THREAD

**Issue by ruben-alonso** [JiscPER/jper#65]
Hi,

I imagine that you missed to remove this part from the octopus when you were doing the cleaning of it for this project. I have found that the notification models are duplicated in jper and the octopus. service/models/notifications and octopus/modules/jper/models define the same objects having more information the one in JPER.  
Thanks

**Comment by richard-jones** [comment]
No, that's deliberate.  The ones in octopus are shared client model objects, not core system objects - they should be allowed to vary independently of the system core.  The reason they are different, is that they have different purposes/scope and are to be used in different, non-overlapping contexts.

**Comment by ruben-alonso** [comment]
Ok, I understand that. But, don't you think it will be better if there is a base class with all the information that is shared among these models and then you create two classes that extend the base one, one for the client and one for the core?
